packages/backend-core/tests/core/utilities/utils/queue.ts

```diff
@@ -0,0 +1,31 @@
+import { Queue } from "bull"
+import { GenericContainer, Wait } from "testcontainers"
+
+export async function useRealQueues() {
+  const redis = await new GenericContainer("redis")
+    .withExposedPorts(6379)
+    .withEnvironment({})
+    .withLabels({ "com.budibase": "true" })
+    .withReuse()
+    .withWaitStrategy(
+      Wait.forSuccessfulCommand(
+        `until redis-cli  ping | grep -q PONG; do
+          echo "Waiting for Redis to be ready..."
+          sleep 1
+        done
```

Comment:
I don't think we should loop here. `Wait.forSuccessfulCommand` already implements its own loop to retry this command.